packages/util/src/types.ts

```diff
@@ -40,3 +40,5 @@ export type BnList = {
   0: BN;
   1: BN;
 } & BN[];
+
+export type Time = [number, number, number, number];
```

Comment:
Maybe be explicit in a struct rather than a tuple? Can just see myself having to lookup.